tests/functional/jwt_helper.js

```diff
@@ -8,15 +7,47 @@ const JWE = jose.JWE;
 
 const crypto = require('crypto');
 
-const signingKey = './tests/functional/sdc-user-authentication-signing-rrm-private-key.pem';
-const encryptionKey = './tests/functional/sdc-user-authentication-encryption-sr-public-key.pem';
+const signingKeyString = '-----BEGIN RSA PRIVATE KEY-----\n' +
+  'MIIEogIBAAKCAQEAvZzMraB96Wd1zfHS3vW3z//Nkqz+9HfwViNje2Y5L6m3K/7r\n' +
+  'aA0kUsWD1f6X7/LIJfkCEctCEj9q19+cX30h0pi6IOu92MlIwdH/L6CTuzYnG4PA\n' +
+  'CKT8FZonLw0NYBqh8p4vWS8xtNHNjTWua/FFTlxdtYnEb9HbUZkg7dXAtnikozlE\n' +
```

Comment:
Not a blocker but I think it would be nicer if we used the ES6 backticks `` ` ` `` syntax for multiline strings instead of concatenating the string.

Reply:
Tried it at @ajmaddaford's request, doesn't work with our current setup.